pkg/native/serializers/serializer_cdx.go

```diff
@@ -339,6 +339,28 @@ func (s *SerializerCDX) nodeToComponent(n *sbom.Node) *cdx.Component {
 		}
 	}
 
+	if n.Suppliers != nil && len(n.GetSuppliers()) > 0 {
+		// TODO(degradation): CDX type Component only supports one Supplier while protobom supports multiple
+
+		nodesupplier := n.GetSuppliers()[0]
+		oe := cdx.OrganizationalEntity{
+			Name: nodesupplier.GetName(),
```

Comment:
At some point perhaps we should think about how to translate here when dealing with organizations and persons. I think it is fine to add it like this now but the `sbom.Person` type no only is recursive, it can capture an organization (person.IsOrg) or individual. Perhaps there are better ways to match the cyclonedx field semantics?